.github/actions/e2e/action.yml

```diff
@@ -73,11 +82,28 @@ runs:
     with:
       go-version-file: go.mod
       cache-dependency-path: "**/*.sum"
+  - name: Install replicated CLI for CMX
```

Comment:
we should make a github action for this!

Reply:
why not use the replicated / vendor-cli docker image that's published to do these?